src/components/Header/HeaderLink.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react'
+import styled from 'styled-components'
+import { NavLink, NavLinkProps } from 'react-router-dom'
+import transparentize from 'polished/lib/color/transparentize'
+
+import { ExternalLink } from '../../theme/components'
+
+const StyledNavLink = styled(NavLink)`
+  ${({ theme }) => theme.flexRowNoWrap}
+  align-items: left;
+  border-radius: 3rem;
+  outline: none;
+  cursor: pointer;
+  text-decoration: none;
+  color: ${({ theme }) => theme.text5};
+  width: fit-content;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19.5px;
+  font-family: 'Montserrat';
```

Comment:
Do we have to declare the font-family here, don't we inherit from global styles?

Reply:
Good point! @denciu just moved code. This PR shouldn't address this. Create a follow-up ticket for this.